test/k8sT/KafkaPolicies.go

```diff
@@ -107,157 +106,112 @@ var _ = Describe("K8sValidatedKafkaPolicyTest", func() {
 	})
 
 	AfterEach(func() {
-		By("Deleting demo path")
+		kubectl.Delete(l7Policy)
+	})
+
+	AfterAll(func() {
 		kubectl.Delete(demoPath)
 		err := kubectl.WaitCleanAllTerminatingPods()
 		Expect(err).To(BeNil(), "Terminating containers are not deleted after timeout")
-
 	})
 
 	It("KafkaPolicies", func() {
-		// GH-3440 Re-enable when stable
-		return
-
-		clusterIP, err := kubectl.Get(helpers.DefaultNamespace, "svc").Filter(
-			"{.items[?(@.metadata.name == \"kafka-service\")].spec.clusterIP}")
-		logger.Infof("KafkaPolicyRulesTest: cluster service ip '%s'", clusterIP)
-		Expect(err).Should(BeNil())
-
-		By("Waiting for all Cilium Pods and endpoints to be ready ")
-		By("Waiting for node K8s1")
-		ciliumPod1, _ := kubectl.WaitCiliumEndpointReady(podFilter, helpers.K8s1)
```

Comment:
@eloy: This is a multimode setup, with 

 ```
Kafka multinode setup:
 
 			K8s1:
 				1. empire-backup
 				2. empire-hq
 				3. kafka-broker : ingress policy only
 
 			K8s2:
 			   1. zook
 			   2. empire-outpost-8888
 			   3. empire-outpost-9999
```
 	

You are only testing for k8s2 nodes and have removed the logic for making sure all pods are up on k8s1